src/main/java/de/bsi/secvisogram/csaf_cms_backend/rest/AdvisoryController.java

```diff
@@ -200,9 +346,46 @@ public ResponseEntity<EntityCreateResponse> importCsafDocument(@RequestBody Json
      * @return response with the new optimistic locking revision
      */
     @PatchMapping("/{advisoryId}")
-    @Operation(summary = "Change advisory.", tags = {"Advisory"},
-            description = "Change a CSAF document in the system. On saving a document its content (version) may change " +
-                          " Thus, after changing a document, it must be reloaded on the client side.")
+    @Operation(summary = "Change advisory.", 
+               tags = {"Advisory"},
+               description = "Change a CSAF document in the system. On saving "
+                   + "a document its content (version) may change. Thus, after "
+                   + "changing a document, it must be reloaded on the client"
+                   + " side.")
+    @ApiResponses(value = {
+        @ApiResponse(
+          responseCode = "200", 
+          description = "Id and revison id of new advisory-", 
+          content = { 
+            @Content(
+              mediaType = MediaType.APPLICATION_JSON_VALUE,
+              schema = @Schema(
+                  implementation = EntityUpdateResponse.class
+              )
+            )
+          }
+        ),
+        @ApiResponse(
+            responseCode = "400", 
+            description = "Database error."
```

Comment:
Database error? vs. Bad Request?

Reply:
Clarified description. It's not simply a database error. It is a malformed submitted document.